WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -172,7 +172,9 @@ extension PointOfSaleAggregateModel {
 
     func startNewCart() {
         removeAllItemsFromCart()
-        orderController.clearOrder()
+        Task {
+            await orderController.clearOrder()
```

Comment:
Check if we don't need to capture orderController weakly here. It likely doesn't cause memory leaks but we can check just to be safe. I remember it did for some other calls that didn't complete.